### PR TITLE
Imp: use bytes pool in session.go

### PR DESCRIPTION
**Pull request by AlexStocks**
five modifications:

Imp: use bytes pool in session.go;
Imp: use bytes.Bytes from memory pool
Fix: taskPoll -> taskPool;
Add: http pprof;
Add: task pool for client;

### CODE REVIEW

File: session.go [hunk 1]
```diff
@@ -413,16 +414,20 @@ func (s *session) WriteBytesArray(pkgs ...[]byte) error {
 	var (
 		l      int
 		err    error
-		length uint32
+		length int
+		arrp   *[]byte
 		arr    []byte
 	)
-	length = 0
+	length = 64
```
[the remark this answers is not in the document]
Comment (AlexStocks) [reply session.go]: a little longer(64iB) than its real length.

**Comment by wongoo** [comment]
LGTM

**Comment by gaoxinge** [comment]
LGTM.

File: client.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ import (
 )
 
 import (
+	gxbytes "github.com/divebomb/gost/bytes"
```
[the remark this answers is not in the document]
Comment (AlexStocks) [reply client.go]: done

**Comment by wongoo** [comment]
@AlexStocks  import path error: `github.com/divebomb/gost/bytes`

**Comment by AlexStocks** [comment]
@wongoo fix it.

**Comment by aliiohs** [comment]
LGTM

**Pull request merged**